frontend/sw360-portlet/src/main/webapp/html/components/detail.jsp

```diff
@@ -35,6 +36,13 @@
     <portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/>
 </portlet:resourceURL>
 
+<portlet:actionURL var="updateComponentURL" name="updateComponent">
+    <portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/>
+</portlet:actionURL>
+
+<c:set var="autocompleteUrl" value="<%=PortalConstants.CODESCOOP_URL%>"/>
```

Comment:
`autocompleteUrl` is a rather generic name for such a spezific variable

Reply:
renamed

Reply:
@maxhbr unfortunately I can not escape from s:set var in jsp eval...